### PR TITLE
Adding script to verify commit message

### LINKED ISSUE #140: Adding commit message checker
For the quality of every commit message, please add the Commit Message checker just like [1].

[1] https://github.com/vuejs/vue-next/blob/master/scripts/verifyCommit.js

### PR DESCRIPTION
**Pull request by sshniro**
Resolves #140

### CODE REVIEW

**Comment by juzhiyuan** [comment]
Thanks! I will have a try today.

**Comment by juzhiyuan** [comment]
Thanks, and please add the License header to this file, for we are referencing this file from `https://github.com/vuejs/vue-next/blob/master/scripts/verifyCommit.js`

File: scripts/verifyCommit.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Invoked on the commit-msg git hook by yorkie.
```
Comment (juzhiyuan) [inline scripts/verifyCommit.js]: There should be the same as the origin repo.

File: scripts/verifyCommit.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+const chalk = require('chalk');
+const msgPath = process.env.GIT_PARAMS;
```
Comment (juzhiyuan) [inline scripts/verifyCommit.js]: `msgPath` is undefined on my MacOS, please take a look.

Comment (sshniro) [reply scripts/verifyCommit.js]: sure @juzhiyuan I will check this.

**Comment by juzhiyuan** [comment]
Any update?

**Comment by sshniro** [comment]
Hi @juzhiyuan , due to the work from home situation I could not get my hands on to a mac to resolve this issue. Shall we merge this after the license header update and keep a separate issue for mac?

**Comment by juzhiyuan** [comment]
Sorry we can't merge it because it doesn't work on my Mac, but don't worry, just take your time :) We can continue on this when the virus got away.

**Comment by sshniro** [comment]
Okay @juzhiyuan I'm currently closing the PR so anyone wishes are open to fixing this. I will try to attempt another issue from the backlog.

**Comment by juzhiyuan** [comment]
Ok

**Pull request closed without merging**